run-tests.sh

```diff
@@ -0,0 +1,3 @@
+check-manifest --ignore ".drone.yml,.readthedocs.yml" && \
```

Comment:
Please include the ``pydocstyle`` and ``isort`` checks.

Reply:
The `pydocstyle` check could be included in #39  issue, because the number of files changed in pull request